ibm/service/power/data_source_ibm_pi_volume.go

```diff
@@ -145,6 +150,7 @@ func dataSourceIBMPIVolumeRead(ctx context.Context, d *schema.ResourceData, meta
 	d.Set("primary_role", volumedata.PrimaryRole)
 	d.Set("auxiliary_volume_name", volumedata.AuxVolumeName)
 	d.Set("master_volume_name", volumedata.MasterVolumeName)
+	d.Set("io_throttle_rate", volumedata.IoThrottleRate)
```

Comment:
Is it set for all the volumes by default? If not then this test fail for some volumes.

Reply:
You’re correct. I update the test case